packages/EasyStandard/readme.md

```diff
@@ -18,6 +18,48 @@ We recommend to use [Composer][1] to manage your dependencies. You can require t
 $ composer require --dev eonx/easy-standard
 ```
 
+### Prepare configuration file
+
+You can use one of next the names for configuration: `easy-coding-standard.yml`, `easy-coding-standard.yaml`, `ecs.yml`, `ecs.yaml` or `ecs.php`. Create this file in root folder of project.
```

Comment:
`...one of the following names...`,   `...in the root folder of the project.`

Reply:
Done.